domains/vtb.json

```
@@ -4,6 +4,7 @@
     "email": "kimgiahuy@proton.me"
   },
   "records": {
-    "URL": "https://github.com/ntbeastboy"
+    "URL": "https://github.com/ntbeastboy",
+    "TXT": "dh=fc7a1e36282db74ddb6cfd5a18496dea55429930"
```

add the TXT record in new json file "_discord.vtb.json"

```suggestion
    "URL": "https://github.com/ntbeastboy"
```